charts/gafaelfawr/templates/serviceaccount-tokens.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.tokens.serviceAccount.create .Values.tokens.secrets -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "gafaelfawr.tokensServiceAccountName" . }}
+  labels:
+    {{- include "gafaelfawr.labels" . | nindent 4 }}
+  annotations:
+    iam.gke.io/gcp-service-account: {{ required "cloudsql.serviceAccount must be set to a valid Google service account" .Values.cloudsql.serviceAccount | quote }}
```

Comment:
is this only used for cloudsql?  Might want to gate that even if it's not used so it's more clear that's what it's used for

Reply:
Yes, good catch, I cut and pasted that from the other service account that gates at the top level and didn't notice that. Thank you!